e2e/test/scenarios/metrics/browse.cy.spec.ts

```diff
@@ -1,13 +1,15 @@
 const { H } = cy;
+import { USERS } from "e2e/support/cypress_data";
 import { SAMPLE_DATABASE } from "e2e/support/cypress_sample_database";
 import {
   FIRST_COLLECTION_ID,
   ORDERS_MODEL_ID,
 } from "e2e/support/cypress_sample_instance_data";
+import type { StructuredQuestionDetails } from "e2e/support/helpers";
 
 const { ORDERS_ID, ORDERS, PRODUCTS_ID, PRODUCTS } = SAMPLE_DATABASE;
 
-type StructuredQuestionDetailsWithName = H.StructuredQuestionDetails & {
+type StructuredQuestionDetailsWithName = StructuredQuestionDetails & {
```

Comment:
Not sure how was this passing the linter and the CI in the past?
<img width="597" alt="image" src="https://github.com/user-attachments/assets/19c00c7a-0bf0-4a9e-b09e-90b00a8c499d" />